domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/response/TimelineEventValidationTest.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.cmc.domain.models.response;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Test;
+import uk.gov.hmcts.cmc.domain.models.sampledata.response.SampleTimelineEvent;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.cmc.domain.utils.BeanValidator.validate;
+
+public class TimelineEventValidationTest {
+
+    @Test
+    public void passesForValidSample() {
+        //given
+        TimelineEvent timelineEvent = SampleTimelineEvent.validDefaults();
+        //when
+        Set<String> response = validate(timelineEvent);
+        //then
+        assertThat(response).isEmpty();
+    }
+
+    @Test
+    public void failsWhenDescriptionEmpty() {
+        //given
+        TimelineEvent timelineEvent = SampleTimelineEvent.builder()
+            .withDescription(null)
```

Comment:
That does not test description is empty, it tests description is missing. I guess we could have one test case checking `required not null` rule and the few more to test more specific validation rules.